perfkitbenchmarker/linux_benchmarks/netperf_benchmark.py

```diff
@@ -396,6 +404,10 @@ def RunNetperf(vm, benchmark_name, server_ip, num_streams):
                         thinktime_array_size=FLAGS.netperf_thinktime_array_size,
                         thinktime_run_length=FLAGS.netperf_thinktime_run_length)
 
+  if FLAGS.netperf_mss and 'TCP' in benchmark_name.upper():
+    netperf_cmd += (' -G {mss}b'.format(
+                      mss=FLAGS.netperf_mss))
```

Comment:
Actually now that I think about it more, let's go ahead and rename the effective value `netperf_mss` since that's what you'll want for analysis.  And either rename the requested value `netperf_mss_requested` or (and?) log a warning if they don't match. How often do we expect them not to match?

Reply:
In my testing, the reported effective MSS is always 12 bytes smaller than the requested MSS. This is likely due to the size of the options carried on the data segment. 

In the case that a MSS is requested that is larger than the MTU, the default MSS will be used and that is what is reported as the effective MSS.